oeps/oep-0002.rst

```diff
@@ -69,6 +70,32 @@ following fields:
     If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be
     ``False``, unless the reporting tool can auto-detect accordance.
 
+``track-pulls``: boolean (optional)
+
+    .. This should be moved to a separate OEP
+
+    Whether or not this repository should be tracked for creation of OSPR tickets.
```

Comment:
Ah, I thought that was there because we didn't want the bot commenting on every repo.